pupil.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env python
+"""
+Pupil inherits from Distorted and has properties *opd* and *name*. Its methods are *_compute_pupil*, *_path_diff*
+"""
+
+import glob
+import numpy as np
+
+from distorted import Distorted
+from aperture import Aperture
+from optiarray import OpticalArray
+import zernike as ze
+
+
+class Pupil(Distorted):
```

Comment:
I'm just curious:  Why did you put all these into separate files?  I would have put all the sub-classes into a single file so I can have them all imported together in a single import statement... you choose a different layout, so I'm curious.